mutis/signal.py

```diff
@@ -109,35 +118,44 @@ def OU_fit(self, bins=None, rang=None, a=1e-5, b=100):
             # print('pcov: ')
             # print(np.sqrt(np.diag(pcov)))
             x_c = np.linspace(1e-5, 1.1 * np.max(x), 1000)
-            plt.plot(x_c, self.pdf(x_c, *popt), "k-", label="curve_fit", alpha=0.8)
-            res["curve_fit"] = (popt, np.sqrt(np.diag(pcov)))
+            plt.plot(x_c, self.pdf(x_c, *popt), 'k-', label='curve_fit', alpha=0.8)
+            res['curve_fit'] = (popt, np.sqrt(np.diag(pcov)))
```

Comment:
Ah, I forgot about the double quotes. I used  `y`, `t,` `dy`, `dt` because they are used extensively over the functions and makes reading the code easier than `self.values`, `self.times,` `np.diff(self.values)`, `np.diff(self.times)` but they can be changed back, it does not affect the result.

Reply:
Ok, I guess it is just a matter of taste